presto-spi/src/main/java/com/facebook/presto/spi/PrestoSparkExecutionContext.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi;
```

Comment:
Let's move this to a new package `com.facebook.presto.spi.prestospark` or something similar.  If we create that new package, I would also recommend putting the physical resource settings there so we don't need the inner class below, we can just directly annotate the original class.